wordpress-mobile/WordPress-iOS
# UL&S: Implements re-authentication flow tracking.

WPAuth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/489

Fixes an issue with tracking during re-authentication.
Implements the re-authentication flow tracking.

## Testing:

Test 1:

1. Log in to a WP.com account
2. Using the web, disconnect WPiOS. (https://wordpress.com/me/security/connected-applications)
3. Do something in WPiOS that would trigger a network request.
4. Once the password screen is up check.

🔵 Tracked: unified_login_step <flow: login_password, source: reauthentication, step: start>

5. Tap on Help and verify:

🔵 Tracked: unified_login_interaction <click: show_help, flow: login_password, source: reauthentication, step: start>
🔵 Tracked: unified_login_step <flow: login_password, source: reauthentication, step: help>

6. Tap “Close” and verify:

🔵 Tracked: unified_login_interaction <click: dismiss, flow: login_password, source: reauthentication, step: help>

7. Enter the correct password, tap “Continue” and verify:

🔵 Tracked: unified_login_interaction <click: submit, flow: login_password, source: reauthentication, step: start>
🔵 Tracked: unified_login_step <flow: login_password, source: reauthentication, step: success>

## PR submission checklist:
